turtlebot3_example/turtlebot3_example/turtlebot3_obstacle_detection/turtlebot3_obstacle_detection.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+#
+# Copyright 2019 ROBOTIS CO., LTD.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors: Ryan Shim, Gilbert
+
+from geometry_msgs.msg import Twist
+import numpy
```

Comment:
`numpy` is not included ros2 lib.

Reply:
Can you tell me more about this comment? Are you recommending another library instead of numpy?

Reply:
Sorry for my insufficient the comment.
The `numpy` is general library in python3. 
Therefore, you should have import `numpy` before `geometry_msgs` that is general library in ROS.

You can find order of import or include library in robotis code style guide.

Reply:
@routiful 
Thank you.